### ISSUE TITLE
Manager says "Installing..." after install failed

### ISSUE THREAD

**Issue by KobeW50** [ReVanced/revanced-manager#1590]
### Bug description

After an installation fails, it will still say "installing..." at the top of the screen. 

Since the installation is now halted, this message should be changed to "Installation failed" or a message related to the error status, such as "Installation conflict".

Once the user presses the "Install" button again the message should ofc change back to "Installing...".

This video demonstrates how it should appear.

https://github.com/ReVanced/revanced-manager/assets/84587632/5f405ba3-5e96-4417-add4-1fb6a743dd22

Additionally, it should also be added to the patch log that the "Installation failed" if an installation error occurs. 

### Version of ReVanced Manager and version & name of application you tried to patch

1.18.0

### Installation type

None

### Device logs

```shell
NA
```

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The issue is solely related to the ReVanced Manager

**Comment by KobeW50** [comment]
> Since the installation is now halted

I relied on this reasoning when deciding whether it should be a bug report or a feature request. One can disagree and view the entire installation process as a constant "installing..." for the user, in which case this issue should be closed.

**Comment by oSumAtrIX** [comment]
Is this still an issue?

**Comment by KobeW50** [comment]
Yes. Until the app is successfully installed or the user cancels the installation attempts is says "Installing"

**Comment by oSumAtrIX** [comment]
ReVanced Manager will soon be released as [a full rewrite](https://github.com/ReVanced/revanced-manager/tree/compose-dev). Over time, many feature requests have been made for the current version of ReVanced Manager, but to accelerate the development of the new version of ReVanced Manager, we have to focus on it, which I am closing this issue according to the [contribution guidelines](https://github.com/ReVanced/revanced-manager/blob/main/CONTRIBUTING.md#-submitting-a-feature-request):

> Requests can be accepted or rejected at the discretion of maintainers of ReVanced Patches

This feature request has been closed for one or more of the following reasons:

- Stale for too long/ Low priority/ Maintainers un-interested/ Too complex to realize
- Feature planned/ already implemented for the new version of ReVanced Manager
- Already implemented

Regardless, anyone who comes across this feature request can PR it to the current version of ReVanced Manager until the new version is ready. Thanks for your feature request!
